addOns/openapi/src/main/java/org/zaproxy/zap/extension/openapi/generators/BodyGenerator.java

```diff
@@ -77,8 +82,10 @@ public String generate(Schema<?> schema) {
         boolean isArray = schema instanceof ArraySchema;
         LOG.debug("Generate body for object {}", schema.getName());
 
-        if (isArray) {
+        if (schema instanceof ArraySchema) {
```

Comment:
Why this change, or why was not `isArray` removed?

Reply:
I see it was reintroduced when merging changes, so just needs to be removed again.